Assets/Scripts/MQ/MQ.cs

```diff
@@ -153,23 +153,29 @@ public static List<Queue> Parse(_QueueResponseJson queueResponseJson)
                     case "alias":
                         queue = new AliasQueue();
                         ((AliasQueue)queue).targetQueueName = queueJson.alias.targetName;
+                        queue.holdsMessages = false;
```

Comment:
since `holdsMessages` is true for all queues of the same type, I would set it in the specific queue's class with `public const boolean holdsMessages = false`. similarly for other types of queues as well

Reply:
yes.